portfolio.css

```diff
@@ -0,0 +1,308 @@
+	gap: 20px;
+	
+	margin: auto 20px;
+	font-size: 2.5rem;
+}
+
+
+/* --- HEADER ---------------------- */
+
+header {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 20px;
+	align-items: center;
+	
+	background-color: #26315f;
+	height: 100%;
+	padding: 40px;	
+	margin-top: 60px;
+}
+
+.tardi-image {
+	height: 150px;
+	margin-right: 5px;
+	border-radius: 10px;
+}
+.header-text {
+	max-width: 400px;
+	font-size: 3rem;
+}
+
+h1 {
+	font-size: 4rem;
+}
+
+
+/* --- ABOUT --------------------- */
+
+.about {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 20px;
+	align-items: center;
+	
+	height: 100%;
+	background-color: #20396a;
+	padding: 40px;
+
+	scroll-margin: 60px;
+}
+
+.profile-pic {
+	height: 150px;
+	border-radius: 50px;
+}
+
+.about-text {
+	font-size: 2rem;
+	max-width: 500px;
+	
+	text-shadow: 0 0 2px #26315f;
+}
+
+
+/* --- PORTFOLIO ------------------- */
+
+.portfolio {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	justify-content: center;
+	
+	width: 100%;
+	height: 100%;
+	padding: 40px;
+
+	background-color: #27457e;
+	font-size: 2rem;
+	
+	scroll-margin: 60px;
+	
+	text-shadow: 0 0 2px #26315f;
+	
+}
+
+.portfolio-intro {
+	min-width: 300px;
+	max-width: 500px;
+	margin: 0 auto;
+}
+
+.projects-container {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	gap: 20px;
+	align-items: center;
+	
+	padding: 40px;
+	width: 100%;
+}
+
+.project {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: center;
+	gap: 30px;
+	align-items: center;
+	
+	width: 90%;
+	border: 2px solid #9a4a32;
+	border-radius: 10px;
+	
+	max-width: 800px;
+	padding: 20px;
+}
+
+.project-links {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	justify-content: right;
+	gap: 20px;
+	align-items: center;
+}
+
+.thumbnail {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.project-image {
+	height: 90px;
+	border: 2px solid #a55031;
+	border-radius: 10px;
+	margin-top: 10px;
+}
+
+.project-text {
+	width: 60%;
+}
+
+
+/* --- CONTACT ------------------------*/
+
+.contact {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 20px;
+	align-items: center;
+	
+	width: 100%;
+	height: 100%;
+	padding: 40px;
+
+	background-color: #9a4a32;
+	color: #5d3c7f;
+	font-size: 2rem;
+	
+	scroll-margin: 60px;
+}
+
+.contact-text {
+	max-width: 400px;
+}
+
+
+/* --- FOOTER --------------------------*/
+
+footer {
+	height: 50px;
+	background-color: #184e96;
+	padding: 10px;
+	
+	font-size: 2rem;
+	margin: auto 0;
+}
+
+
+/* --- RESPONSIVENESS ---------------------*/
+
+@media screen and (max-width: 800px) {
+
+	nav {
+		justify-content: space-evenly;
+	}
+
+	header {
+		flex-direction: column;
+		padding: 40px;
+	}
+	
+	.header-text {
+		margin: 0 auto;
+	}
+
+	.about {
+		flex-direction: column;
+	}
+
+	.portfolio {
+		flex-direction: column;
+
+	}
+	
+	.project {
+		flex-direction: column;
+		gap: 10px;
+	}
+	
+	.project-text {
+		width: 80%;
```

Comment:
i would remove this as it looks similar enough without (provided you've removed the width from my previous comment as well). if you want it to have space on the sides, that should be margins or padding instead